blinka.html

```diff
@@ -6,6 +6,9 @@
 
 <div id="downloads-page">
   <h1>Blinka</h1>
+  <p>
+    Blinka brings CircuitPython APIs and, therefore, libraries to single board computers (SBCs). It is a <a href="https://pypi.org/project/Adafruit-Blinka/">pip installable Python library</a> that runs in normal "desktop" Python. The CircuitPython runtime isn't used. <a href="https://pypi.org/search/?q=adafruit-circuitpython">Libraries</a> can also be installed via pip. See <a href="https://learn.adafruit.com/circuitpython-on-raspberrypi-linux"> the guide</a> for further details.
```

Comment:
Consider including `CircuitPython` before libraries. I had to read the sentence a couple of times, and that would clarify things. Including "CircuitPython" in the later "Libraries" sentence might make sense as well. Up to you though! 
```suggestion
    Blinka brings CircuitPython APIs and, therefore, CircuitPython libraries to single board computers (SBCs). It is a <a href="https://pypi.org/project/Adafruit-Blinka/">pip installable Python library</a> that runs in normal "desktop" Python. The CircuitPython runtime isn't used. <a href="https://pypi.org/search/?q=adafruit-circuitpython">CircuitPython libraries</a> can also be installed via pip. See <a href="https://learn.adafruit.com/circuitpython-on-raspberrypi-linux"> the guide</a> for further details.
```